packages/web-components/fast-components/src/breadcrumb/fixtures/base.html

```diff
@@ -0,0 +1,178 @@
+<fast-design-system-provider use-defaults>
+    <h1>Breadcrumb</h1>
```

Comment:
Do you have a scenarios folder and index.html file? I think this changed just yesterday, here's an example for radio-group:
![Screen Shot 2020-10-16 at 10 12 21 AM](https://user-images.githubusercontent.com/25805778/96288483-5cb8e980-0f98-11eb-8554-0476e8682c02.png)

Reply:
thank you. I added a scenario file.